README.md

```diff
@@ -23,22 +23,43 @@ s3_prefix=prefix/
 
 ### Run
 
+#### pre-Java 9 versions
+
 ```bash
 java -jar record-and-upload-VERSION-capsule.jar --config .private/aws-test-secrets --store ./build/play
 ```
 
+#### Java 9 versions
+
+```bash
+java --illegal-access=warn --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED -jar record-and-upload-VERSION-capsule.jar --config .private/aws-test-secrets --store ./build/play
+```
+
 
 ## Development
 
 ### Build and run as command-line app
 
 This will grate a maven based Jar that will download the required dependencies before running the app:
+
+#### Running and building with pre-Java 9 versions
+
 ```bash
 ./gradlew clean mavenCapsule
 rm -R ~/.capsule/deps/ro
 java -jar ./build/libs/record-and-upload-`cat version.txt`-capsule.jar --config .private/aws-test-secrets --store ./build/play
 ```
 
+#### Running and building with Java 9
+
+Ensure `JAVA_HOME` points to the Java 9 SDK home folder. Use `java -version` to check if the java launcher on the `PATH` is version 9.0 (aka 1.9) or higher.
+
+```bash
+./gradlew clean mavenCapsule
+rm -R ~/.capsule/deps/ro
+java --illegal-access=warn --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED -jar ./build/libs/record-and-upload-`cat version.txt`-capsule.jar --config .private/aws-test-secrets --store ./build/play
```

Comment:
This command failed to work for me:
```
➜  rec.test /Library/Java/JavaVirtualMachines/jdk-9.0.1.jdk/Contents/Home/bin/java --illegal-access=warn --add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.lang.invoke=ALL-UNNAMED -jar record-and-upload-0.0.10-capsule.jar  --config credentials.config --store ./store
WARNING: Illegal reflective access by Capsule (file:/private/tmp/rec.test/record-and-upload-0.0.10-capsule.jar) to field com.sun.jmx.mbeanserver.JmxMBeanServer.mbsInterceptor
INFO  [main]       - Starting recording app
INFO  [main]       - Checking permissions
Exception in thread "main" java.lang.NoClassDefFoundError: javax/xml/bind/JAXBException
```

This is what worked for me:
```
/Library/Java/JavaVirtualMachines/jdk-9.0.1.jdk/Contents/Home/bin/java --illegal-access=warn "--add-modules=java.xml.bind,java.activation" -jar record-and-upload-0.0.10-capsule.jar  --config credentials.config --store ./store
```

Reply:
That looks about right for the error message you got, mine wouldn't go any further than trying to connect with the S3 bucket (which I have access to now), so will give it a whirl to see if there are any more such exceptions lurking about

Reply:
This now works for me as well after AWS S3 credentials, I see this output:
```
WARNING: Illegal reflective access by Capsule (file:/home/satyasai/git-repos/BeFaster/record-and-upload/build/libs/record-and-upload-0.0.11-SNAPSHOT-capsule.jar) to field com.sun.jmx.mbeanserver.JmxMBeanServer.mbsInterceptor
WARNING: Illegal reflective access by Capsule (file:/home/satyasai/git-repos/BeFaster/record-and-upload/build/libs/record-and-upload-0.0.11-SNAPSHOT-capsule.jar) to field java.lang.ProcessImpl.pid
INFO  [main]       - Starting recording app
INFO  [main]       - Checking permissions
Java HotSpot(TM) 64-Bit Server VM warning: You have loaded library /tmp/humble9670263996444646425.tmp which might have disabled stack guard. The VM will try to fix the stack guard now.
It's highly recommended that you fix the library with 'execstack -c <libfile>', or link it with '-z noexecstack'.
INFO  [VideoRec]   - Open the input stream
INFO  [Upload]     - Sync local files with remote
INFO  [main]       - Logging initialized @2610ms to org.eclipse.jetty.util.log.Slf4jLog
INFO  [main]       - jetty-9.4.7.v20170914
INFO  [Upload]     - Uploading file build/play/record-and-upload-20171208T221336.log
INFO  [main]       - Started ServerConnector@4a7a965d{HTTP/1.1,[http/1.1]}{localhost:41375}
INFO  [main]       - Started @2691ms
INFO  [SourceRec]  - Start recording to "sourcecode_20171208T221516.srcs"
INFO  [Upload]     - Finished uploading file build/play/record-and-upload-20171208T221336.log
INFO  [Upload]     - Uploading file build/play/sourcecode_20171208T221516.srcs
INFO  [Upload]     - Finished uploading file build/play/sourcecode_20171208T221516.srcs
INFO  [Upload]     - Uploading file build/play/record-and-upload-20171208T221242.log
INFO  [VideoRec]   - Start recording to "screencast_20171208T221516.mp4" at 16 fps with 4 screenshots/sec
INFO  [Upload]     - Finished uploading file build/play/record-and-upload-20171208T221242.log
INFO  [Upload]     - Uploading file build/play/record-and-upload-20171208T221515.log
INFO  [Upload]     - Finished uploading file build/play/record-and-upload-20171208T221515.log
INFO  [Metrics]    - Recorded 0h00m01s,   5 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m03s,  13 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m05s,  21 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m07s,  29 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m09s,  37 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m11s,  45 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m13s,  53 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m15s,  61 frames, 0.00 MB |  1 snapshot, 217 ms/snap
INFO  [Metrics]    - Recorded 0h00m17s,  69 frames, 0.00 MB |  1 snapshot, 217 ms/snap
<--snipped-->
```